mhkit/dolfyn/velocity.py

```diff
@@ -913,10 +968,9 @@ def power_spectral_density(
         window : string or array
           Specify the window function.
           Options: 1, None, 'hann', 'hamm'
-        noise : float or array-like
-          A vector of the noise levels of the velocity data with
-          the same first dimension as the velocity vector.
-          Default = 0.
+        noise : numeric or array
+          Instrument noise level in same units as velocity.
+          Default: 0 (ADCP) or [0, 0, 0] (ADV).
```

Comment:
The actual default is None

Reply:
Hmmm, yes changing that to 0 is better for code transparency